src/test/ui/parser/unicode-chars.stderr

```diff
@@ -9,5 +9,17 @@ help: Unicode character ';' (Greek Question Mark) looks like ';' (Semicolon), b
 LL |     let y = 0;
    |              ~
 
-error: aborting due to previous error
+error: unknown start of token: \u{a0}
+  --> $DIR/unicode-chars.rs:5:5
+   |
+LL |         let x = 0;
+   |     ^
+   |
+   = note: character appears 3 more times
+help: Unicode character ' ' (No-Break Space) looks like ' ' (Space), but it is not
+   |
+LL |         let x = 0;
+   |     +
```

Comment:
This is a suggestion diagnostic, right? One that suggests to replace the first no-break space with a normal space? Do you think it's feasible to expand both the primary span and the replacement span from one character to in this case three characters? But first, let's hear what others have to say :)